securedrop/journalist_templates/col.html

```diff
@@ -78,7 +76,7 @@
         {% set modal_data = {
                               "modal_id": "delete-selected-confirmation-modal",
                               "modal_header": gettext('Delete Confirmation'),
-                              "modal_body": gettext('Are you sure you want to delete the selected documents?'),
+                              "modal_body": gettext('Are you sure you want to delete the selected messages and/or files?'),
```

Comment:
This is for consistency with "delete messages and files" on the All sources view deletion?

Reply:
Aye, as per the strings spreadsheet.

Reply:
Updated to "submissions" as per comment below.